pkg/exporter/grpc_proto.go

```diff
@@ -21,19 +24,23 @@ type GRPCProto struct {
 	// maxFlowsPerMessage limits the maximum number of flows per GRPC message.
 	// If a message contains more flows than this number, the GRPC message will be split into
 	// multiple messages.
-	maxFlowsPerMessage int
+	maxFlowsPerMessage            int
+	numberOfRecordsReceivedByGRPC prometheus.Counter
+	errExportByGRPC               prometheus.Counter
 }
 
-func StartGRPCProto(hostIP string, hostPort int, maxFlowsPerMessage int) (*GRPCProto, error) {
+func StartGRPCProto(hostIP string, hostPort int, maxFlowsPerMessage int, m *metrics.Metrics) (*GRPCProto, error) {
 	clientConn, err := grpc.ConnectClient(hostIP, hostPort)
 	if err != nil {
 		return nil, err
 	}
 	return &GRPCProto{
-		hostIP:             hostIP,
-		hostPort:           hostPort,
-		clientConn:         clientConn,
-		maxFlowsPerMessage: maxFlowsPerMessage,
+		hostIP:                        hostIP,
+		hostPort:                      hostPort,
+		clientConn:                    clientConn,
+		maxFlowsPerMessage:            maxFlowsPerMessage,
+		numberOfRecordsReceivedByGRPC: m.CreateNumberOfRecordsReceivedByGRPC("number_of_records_received_by_grpc"),
+		errExportByGRPC:               m.CreateErrorCanNotWriteToGRPC("err_export_by_grpc"),
```

Comment:
Here also I think we could have a single metric, with the appropriate label, for reporting errors, rather than one different metric per exporter